jefferson/cli_options.py

```diff
@@ -0,0 +1,17 @@
+import functools
```

Comment:
There will be no other CLI tool that will use this function, therefore it's an unnecessary abstraction. This module is not needed, the option can be directly applied to the main function.

Reply:
Makes sense. Removed.